v1.2/create-sequence.md

```diff
@@ -4,13 +4,15 @@ summary:
 toc: false
 ---
 
-<span class="version-tag">New in v1.2:</span> The `CREATE SEQUENCE` [statement](sql-statements.html)...
+<span class="version-tag">New in v2.0:</span> The `CREATE SEQUENCE` [statement](sql-statements.html) creates a new sequence in a database. Use a sequence to auto-increment integers in a table.
+
+{{site.data.alerts.callout_info}}Using sequences may slow down your application. Use the serial data type unless a sequence is preferred or required.{{site.data.alerts.end}}
```

Comment:
nit: maybe add "may slow down your application _because it does a write to persistent storage_" or similar, just to be clear.

Reply:
added.